sambaplayersdk/src/main/java/com/sambatech/player/cast/CastDRM.java

```diff
@@ -39,11 +40,20 @@ public void setTicket(String ticket) {
         this.ticket = ticket;
     }
 
+    public void setToken(String token) {
+        this.token = token;
+    }
+
+    public String getToken() {
+        return token;
+    }
+
     @Override
     public String toString() {
         return "{" +
                 "\"SessionId\":\"" + sessionId + '\"' +
                 ", \"Ticket\":\"" + ticket + '\"' +
+                (token != null ? ", \"Token\":\"" + token + '\"' : "") +
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add parentheses to make the operator precedence explicit. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.sambatech.com/coding_rules#rule_key=squid%3AS864)